### PR TITLE
replace all instances of deprecated name

### PR DESCRIPTION
**Pull request by cbudz**
Have removed all instances of Red Hat Access Insights and replaced with Red Hat Insights.

### CODE REVIEW

File: doc-General_Configuration/topics/RH_Access_Insights.adoc [hunk 1]
```diff
@@ -1,31 +1,31 @@
 
 [[rh-access-insights]]
-== Red Hat Access Insights
+== Red Hat Insights
 
-*Red Hat Access Insights* is a new service that uses the collective knowledge to help end users proactively diagnose systems and avoid critical downtime situations. Red Hat Access Insights does this by having systems periodically check in similar to Red Hat Subscription Management.
+*Red Hat Insights* is a new service that uses the collective knowledge to help end users proactively diagnose systems and avoid critical downtime situations. Red Hat Insights does this by having systems periodically check in similar to Red Hat Subscription Management.
```
Comment (dayleparker) [inline doc-General_Configuration/topics/RH_Access_Insights.adoc]: @cbudz Do you think Red Had Insights is old enough now that we should remove the word "new"? Might also future-proof our docs a bit as well.

**Comment by dayleparker** [comment]
@cbudz Hey Chris,
The changes look good to me and I did a quick ctrl+F to check there were no lingering instances of "Access Insights". Only thing I'm wondering if we should also fix/investigate while we're in here anyway, is calling it a "new" feature, and whether it's still tech preview. (I added a comment about this). What do you think?
Cheers,
Dayle

**Comment by cbudz** [comment]
Hey Dayle,

I'm all in favor of whatever can reduce future work, so I'll look into your recommendations and update based on what feedback I get.

-Chris

**Comment by cbudz** [comment]
Hey Dayle,

I got some feedback from Rob Williams who initially reported this BZ. He said this should remain as tech preview for the time being. 
I did remove the reference to "new" though since its been around for a bit now. 

-Chris

**Comment by dayleparker** [comment]
Hey Chris,
Thanks so much for looking into it -- and good call on removing the 'new'. 
Merging!
Cheers,
Dayle

**Pull request merged**